### DISCUSSION TITLE
Issues with rendering / Satori

### DISCUSSION THREAD

**Discussion by edwardhorsford** [KiwiKilian/eleventy-plugin-og-image#214] [Q&A]
This is just a note for others, as I've found Satori's rendering to be unreliable. I spent some time yesterday making a good looking template, but have now found it to be unrenderable.

The primary issue is that Satori seems to [break positioning of inline elements](https://github.com/vercel/satori/issues/484), on which my design relies.

I've also found the limited (and somewhat buggy) implementation of css means that what renders well in browser can often not be reproduced.

**Comment by KiwiKilian** [comment]
Yes, Satori/Yoga do not support inline styles. https://github.com/vercel/satori/issues/325 https://github.com/facebook/yoga/issues/968

If your text is not dynamic, did you try to use flex placement? I'm not sure how you want your result to look based on your buggy screenshot.

Otherwise you could also include an SVG inside of your OG template. If all of this is a dealbreaker an approach to generate OG images directly from SVGs might be better. But this comes with other limitations like no flexbox and manual line breaks.

For some inspirations of what's possible see this [discussion](https://github.com/vercel/satori/discussions/173) and the [playground](https://og-playground.vercel.app/) (sadly syntax is not 1:1 compatible with eleventy-plugin-og-image).

I myself found a lot is possible and had no problem to implement what I wanted to achieve. Your mileage may vary.
